pkg/controller/storagecluster/quickstart_constants.go

```diff
@@ -0,0 +1,201 @@
+
+        Did you expand your cluster?
+      failedTaskHelp: This task isn’t verified yet. Try the task again.
+    summary:
+      success: You have expanded the OpenShift Container Storage operator!
+      failed: Try the steps again.
+  - title: Bucket Class Configuration
+    description: |-
+      **Bucket class**
+      Bucket class policy determine the bucket's data location. Its set of policies which would apply to all buckets (OBCs) created with the specific bucket class.
+
+      Data placement capabilities area built as a multi-tier structure, here are the tiers
+
+      **Spread Tier** - list of backing-stores, aggregates the storage of multiple stores.
+
+      **Mirroring Tier** - list of spread-Tiers, async-mirroring to all mirrors, with locality optimization.
+
+      **Create a new Bucket class:**
+
+      1. Go to installed operators page and click one OpenShift Container Storage,
+      2. Go to bucket class tab.
+      3. Click on **Create Bucket Class**
+      4. Follow the wizard steps to  finish creation process.
+
+    review:
+      instructions: |-
+        ####  To verify that you have created bucket class and backing store. 
+        Is the Bucket Class in ready state?
+      failedTaskHelp: This task isn’t verified yet. Try the task again.
+    summary:
+      success: You have successfully created bucket class
+      failed: Try the steps again.
+  conclusion: Congrats, the OpenShift Container Storage operator is ready to use.`
+
+const gettingStartedQS = `
+apiVersion: console.openshift.io/v1
+kind: ConsoleQuickStart
+metadata: 
+ name: "getting-started-ocs"
+spec: 
+ version: 4.7
+ displayName: "Getting started with Openshift Container Storage"
+ durationMinutes: 10
+ icon: data:image/svg+xml;base64,PHN2ZyBlbmFibGUtYmFja2dyb3VuZD0ibmV3IDAgMCAxMDAgMTAwIiBoZWlnaHQ9IjEwMCIgdmlld0JveD0iMCAwIDEwMCAxMDAiIHdpZHRoPSIxMDAiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+PHBhdGggZD0ibTY2LjcgNTUuOGM2LjYgMCAxNi4xLTEuNCAxNi4xLTkuMiAwLS42IDAtMS4yLS4yLTEuOGwtMy45LTE3Yy0uOS0zLjctMS43LTUuNC04LjMtOC43LTUuMS0yLjYtMTYuMi02LjktMTkuNS02LjktMy4xIDAtNCA0LTcuNiA0LTMuNSAwLTYuMS0yLjktOS40LTIuOS0zLjIgMC01LjIgMi4xLTYuOCA2LjYgMCAwLTQuNCAxMi41LTUgMTQuMy0uMS4zLS4xLjctLjEgMSAuMSA0LjcgMTkuMiAyMC42IDQ0LjcgMjAuNm0xNy4xLTZjLjkgNC4zLjkgNC44LjkgNS4zIDAgNy40LTguMyAxMS40LTE5LjEgMTEuNC0yNC42IDAtNDYuMS0xNC40LTQ2LjEtMjMuOSAwLTEuMy4zLTIuNi44LTMuOS04LjkuNS0yMC4zIDIuMS0yMC4zIDEyLjIgMCAxNi41IDM5LjIgMzYuOSA3MC4yIDM2LjkgMjMuOCAwIDI5LjgtMTAuNyAyOS44LTE5LjIgMC02LjctNS44LTE0LjMtMTYuMi0xOC44IiBmaWxsPSIjZWQxYzI0Ii8+PHBhdGggZD0ibTgzLjggNDkuOGMuOSA0LjMuOSA0LjguOSA1LjMgMCA3LjQtOC4zIDExLjQtMTkuMSAxMS40LTI0LjYgMC00Ni4xLTE0LjQtNDYuMS0yMy45IDAtMS4zLjMtMi42LjgtMy45bDEuOS00LjhjLS4xLjMtLjEuNy0uMSAxIDAgNC44IDE5LjEgMjAuNyA0NC43IDIwLjcgNi42IDAgMTYuMS0xLjQgMTYuMS05LjIgMC0uNiAwLTEuMi0uMi0xLjh6IiBmaWxsPSIjMDEwMTAxIi8+PC9zdmc+
+ description: "Learn how to create persistent file and object storage and connect it to your applications."
```

Comment:
"connect it to your application" - can this be reworded?

Reply:
> Learn how to create persistent files, object storage and connect it with your applications.
changed to above.
Let me know if it sounds good enough

Reply:
I think "attached" would be better than "connected".
"Learn how to create persistent, file , block and object storage and attach it your application."